source/slang/slang-syntax.cpp

```diff
@@ -1263,8 +1259,41 @@ Index getFilterCountImpl(const ReflectClassInfo& clsInfo, MemberFilterStyle filt
                 }
             }
         }
+
         return nullptr;
     }
+    
+    Val* _tryLookupConcreteAssociatedTypeFromThisTypeSubst(ASTBuilder* builder, DeclRef<Decl> declRef)
```

Comment:
I wonder if this loop is not needed anymore?

Reply:
Reverted!